docs/get_started/raspbian_setup.md

```diff
@@ -24,52 +28,47 @@ On Rasbian versions Wheezy and later, you need the following dependencies:
 
 - [G++ (4.8 or later)](https://gcc.gnu.org/gcc-4.8/)
 
-Install these dependencies using the following commands:
+Install these dependencies using the following commands in any directory:
 
 ```bash
     sudo apt-get update
     sudo apt-get -y install git cmake build-essential g++-4.8 c++-4.8 liblapack* libblas* libopencv*
 ```
 
-Clone the MXNet source code repository to your computer, using ```git```.
+Clone the MXNet source code repository using the following ```git``` command in your home directory:
 ```bash
     git clone https://github.com/dmlc/mxnet.git --recursive
+    cd mxnet
 ```
 
-If you are not processing images with MxNet on the Pi you can build MxNet without opencv to minimize the size of the compiled library. You can do this by compiling with the following code:
+If you aren't processing images with MXNet on the Raspberry Pi, you can minimize the size of the compiled library by building MXNet without the Open Source Computer Vision (OpenCV) library with the following commands:
 ```bash
-    cd mxnet
-    export ARM=1
     export USE_OPENCV = 0
     make
 ```
 
-Otherwise you can build the full MxNet library: 
+Otherwise, you can build the complete MXNet library with the following command: 
 ```bash
-    cd mxnet
-    export ARM=1
     make
 ```
 
-Whenever you are compiling MXNet for the Pi you need to ensure the "ARM" environment variable is set to 1 (this removes the x86 specific msse flag in the NNVM Submodule's Makefile, making it ARM compatible)
+Executing either of these commands creates a file called ```libmxnet.so``` in the mxnet/lib directory.
 
-Executing either of these commands creates a library called ```libmxnet.so```.
+*Note - If you are getting build errors it is likely you are on an older version of MXNet, and there are x86 specific -msse CFLAGS hardcoded in the project Makefile or one of the submodule Makefiles that need to be manually removed.*
 
-## Install MXNet for Python On Raspberry Pi
+## Install MXNet Python Bindings
 
-To install python bindings run the following commands:
+To install python bindings run the following commands in the MXNet directory:
 
 ```bash
     cd python
     sudo python setup.py install
 ```
 
-You are now ready to run MxNet on your Raspberry Pi! 
+You are now ready to run MXNet on your Raspberry Pi device.
 
-*Note - Keep in mind that loading the entire MXNet library takes up a significant fraction (200MB+) of the Raspberry Pi's limited RAM, so you might have to switch off the GUI and kill any concurrently running processes to free the RAM needed to load training data or large models into memory.*
+*Note - Because the complete MXNet library takes up a significant amount of the Raspberry Pi's limited RAM, when loading training data or large models into memory, you might have to turn off the GUI and terminate running processes to free RAM.*
```

Comment:
**Note**
Because the complete MXNet library takes up a significant amount of the Raspberry Pi's limited RAM, when loading training data or large models into memory, you might have to turn off the GUI and terminate running processes to free RAM.